kubernetes/table_kubernetes_event.go

```diff
@@ -0,0 +1,200 @@
+package kubernetes
+
+import (
+	"context"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+func tableKubernetesEvent(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "kubernetes_event",
+		Description: "Kubernetes Event is a report of an event somewhere in the cluster.",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"name", "namespace"}),
+			Hydrate:    getK8sEvent,
+		},
+		List: &plugin.ListConfig{
+			Hydrate:    listK8sEvents,
+			KeyColumns: getCommonOptionalKeyQuals(),
+		},
+		Columns: k8sCommonColumns([]*plugin.Column{
+			{
+				Name:        "action",
```

Comment:
Could you please rearrange the columns in sorted order? The `json` type columns should be at last.

Reply:
Put the fields output by `kubectl get events` first, and moved the JSON fields to the end. Let me know if you have any suggestions to the new order.